ducktape/mark/_mark.py

```diff
@@ -190,10 +190,33 @@ def __eq__(self, other):
         return super(Parametrize, self).__eq__(other) and self.injected_args == other.injected_args
 
 
+class Env(Mark):
+    def __init__(self, **kwargs):
+        self.injected_args = kwargs
+        self.should_ignore = False
+        for key, value in kwargs.iteritems():
+            if os.environ[key] != value:
+                self.should_ignore = True
+
+    @property
+    def name(self):
+        return "ENV"
+
+    def apply(self, seed_context, context_list):
+        for ctx in context_list:
+            ctx.ingore = ctx.ignore or self.should_ignore
```

Comment:
`ctx.ignore` is probably always false because of the typo on the left hand side.. you have written `ctx.ingore` ..

Reply:
yay! you don't know how long i've been looking at that! FFS!